pkg/providers/instancetype/types.go

```diff
@@ -160,6 +196,10 @@ func computeCapacity(ctx context.Context, info *ec2.InstanceTypeInfo, amiFamily
 		v1alpha1.ResourceAWSNeuron:   *awsNeurons(info),
 		v1alpha1.ResourceHabanaGaudi: *habanaGaudis(info),
 	}
+	if amiFamily.IsWindows() {
```

Comment:
Can we just do a raw type check here rather than having to implement a new method on the interface? Like `if _, ok := amiFamily.(amifamily.Windows); ok {`

Reply:
Sure.
Will follow your team convention.